src/app/bulk-edit/bulk-edit-select/bulk-edit-select.component.ts

```diff
@@ -131,18 +188,111 @@ export class BulkEditSelectComponent implements OnInit, OnDestroy {
     this.onNext.emit();
   }
 
-  private loadAvailableElements(): Observable<DataElementIndexResponse> {
-    return this.resources.dataModel
-      .dataElements(this.context.rootItem.id, { all: true })
-      .pipe(
-        catchError((error) => {
-          this.messageHandler.showError(
-            'There was a problem finding the Data Elements.',
-            error
-          );
-          return EMPTY;
-        })
+  private resetContext() {
+    this.context.childDomainType = null;
+    this.context.childItems = [];
+    this.context.profiles = [];
+    this.contextChanged.emit(this.context);
+  }
+
+  private getSuitableChildDomainTypes(): CatalogueItemDomainTypeOption[] {
+    if (
+      this.context.rootItem.domainType === CatalogueItemDomainType.DataModel
+    ) {
+      return [
+        {
+          domainType: CatalogueItemDomainType.ModelDataType,
+          displayName: 'Data types'
```

Comment:
Capitalised as 'Data Types', 'Data Classes' and 'Data Elements' perhaps?

Reply:
No problem, fixed